CONTRIBUTING.md

```diff
@@ -37,7 +89,16 @@ This is a rough outline of what a contributor's workflow looks like:
 - Make sure your commit messages are in the proper format (see below).
 - Push your changes to a topic branch in your fork of the repository.
 - Make sure the tests pass, and add any new tests as appropriate.
-- Submit a pull request to the original repository.
+- Submit a pull request to the original repository and assign it to yourself.
+- Submit a comment with the sole content "@reviewer PTAL" (please take a look) in GitHub
+  and replace "@reviewer" with the correct recipient.
```

Comment:
I think the less the contributors have to do, the better.

Since we require each commit line to have the format "pkg: action", we create a bot that automatically assigns maintainers to the corresponding packages, and if it's not the correct one the assigned reviewer can reassign.